lib/FullScreenContainer.js

```diff
@@ -175,6 +176,17 @@ export default class FullScreenContainer extends React.Component {
     }
   }
 
+  _onPhotoLongPressed() {
+    const onPhotoLongPressed = this.props.onPhotoLongPress;
+
+    // action behaviour must be implemented by the client
+    // so, call the client method or simply ignore this event
+    if (undefined !== this.props.onPhotoLongPress) {
```

Comment:
how about defining a default noop onPhotoLongPress prop instead of undefined checking? Like `onPhotoLongPress: () => {}`